packages/webdriverio/src/index.js

```diff
@@ -61,6 +61,14 @@ export const remote = async function (params = {}, remoteModifier) {
         )
     }
 
+    instance.addLocatorStrategy = (name, script) => {
```

Comment:
How about we move this function into a utility as follows:
```js
instance.addLocatorStrategy = addLocatorStrategyHandler(instance)
```

the utility then looks like

```js
export function addLocatorStrategyHandler (scope) {
    if (scope.strategies.get(name)) {
        throw new Error(`Strategy ${name} already exists`)
    }

    scope.strategies.set(name, script)
}
```

Reply:
Wouldn't it be more like this?

```js
export function addLocatorStrategyHandler (scope) {
    return (name, script) => {
        if (scope.strategies.get(name)) {
            throw new Error(`Strategy ${name} already exists`)
        }

        scope.strategies.set(name, script)
    }
}
```

Reply:
yes